src/messages/layouts/MessageLayoutContainer.cpp

```diff
@@ -235,13 +235,15 @@ void MessageLayoutContainer::paintElements(QPainter &painter,
     }
 }
 
-void MessageLayoutContainer::paintAnimatedElements(QPainter &painter,
+bool MessageLayoutContainer::paintAnimatedElements(QPainter &painter,
                                                    int yOffset) const
 {
+    bool anyElement = false;
```

Comment:
nit: naming, should include something about animated (maybe anyAnimatedElement?)